src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -899,6 +899,44 @@ private IEnumerable<string> FindLocalizableProperties(XamlType xamlType)
 				type = type.BaseType;
 			}
 		}
+		private bool IsAttachedProperty(INamedTypeSymbol declaringType, string name)
+			=> _isAttachedProperty(declaringType, name);
+
+		private IEnumerable<(INamedTypeSymbol ownerType, string property)> FindLocalizableAttachedProperties(string uid)
+		{
+			foreach (var key in _resourceKeys.Where(k => k.StartsWith(uid + "/")))
+			{
+				// fullKey = $"{uidName}.[using:{ns}]{type}.{memberName}";
+				//
+				// Example:
+				// OpenVideosButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip
+
+				var firstDotIndex = key.IndexOf('/');
+
+				var propertyPath = key.Substring(firstDotIndex + 1);
+
+				const string usingPattern = "[using:";
+
+				if(propertyPath.StartsWith(usingPattern))
+				{
+					var lastDotIndex = propertyPath.LastIndexOf('.');
+
+					var propertyName = propertyPath.Substring(lastDotIndex + 1);
+					var typeName = propertyPath
+						.Substring(usingPattern.Length, lastDotIndex - usingPattern.Length)
+						.Replace("]", ".");
+
+					if(GetType(typeName) is { } typeSymbol)
```

Comment:
It looks like `typeName` here is fully qualified name? In that case, `GetType` is much less performant than `_metadataHelper.GetTypeByFullName`